src/Authentication/App/Controllers/LogoutController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Authentication\App\Controllers;
+
+use Illuminate\Container\Attributes\CurrentUser;
+use Illuminate\Http\Response;
+use Lightit\Authentication\Domain\Actions\LogoutAction;
+use Lightit\Users\Domain\Models\User;
+
+class LogoutController
+{
+    public function __invoke(#[CurrentUser] User $user, LogoutAction $logoutAction): Response
```

Comment:
```suggestion
    public function __invoke(LogoutAction $logoutAction): Response
```
`$user` is not being used, meaning you can remove the injection.